### PR TITLE
Mobile version

### PR DESCRIPTION
**Pull request by omarbabou**
In this PR, I have achieved many things such as:

    -  Understand the use of Grid layout in css
    -  Combining Grid and flexbox together for a better design
    -  Understand how linters work and how to use it correctly
    - Make good message commits by following Gitflow instructions
    - Fix linters error

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -77,6 +77,158 @@ <h4>LET'S CONNECT</h4>
...
+              </h5>
+              <p>
+                Hello I'm a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, 
+                don't hestiate to contact me.
+              </p>
+
+              <div class="social_media">
+                <p>LET'S CONNECT</p>
```
Comment (Munsa1) [inline index.html]: - [x] In line with the project requirements, Kindly change this `p` tag for the "LET'S CONNECT" part to a header tag for semantic purposes.

File: index.html [hunk 2]
```diff
@@ -77,6 +77,158 @@ <h4>LET'S CONNECT</h4>
...
+                <ul>
```
Comment (Munsa1) [inline index.html:171]: - [x] Grouping the social media Icons in a `ul` tag should be done in the following order for readability, and clean code purposes.
image


- So as you can see one `ul` tag can group all your icons unlike what hs been done where each icon is wrapped in its own `ul` tag. Kindly fix it.

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @omarbabou, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
PR Descriptive :heavy_check_mark: 
Linters Oassing :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zainsadaqat — APPROVED** [review]
# APPROVED!!!! 🎉 🎉 🎉 🎉 🎉 🎉
## Hey  @omarbabou 😊 😃 

#### Great Job 😄 👏 You've done it. Your project is complete!  There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 

# Congratulations!!! 🎉 🥳 👏 🙌 🎊 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**